app/uk/gov/hmrc/transitmovementsrouter/models/MovementType.scala

```diff
@@ -23,4 +23,8 @@ case class MovementType(value: String) extends AnyVal
 
 object MovementType {
   implicit lazy val movementTypeFormat: Format[MovementType] = Json.valueFormat[MovementType]
+
+  val Departure = MovementType("departure")
```

Comment:
Why we need to create this two additional variables?  I think we can directly pass string and use case class constructor?

Reply:
First we shouldn't pass Strings, second, I would have preferred a Departure and Arrival object, but then it becomes a refactoring ticket.